packager/VisibilityChecker.cc

```diff
@@ -382,43 +368,44 @@ class VisibilityCheckerPass final {
         }
     }
 
-    void addExportInfo(core::Context ctx, core::ErrorBuilder &e, core::SymbolRef litSymbol) {
+    void addExportInfo(core::Context ctx, core::ErrorBuilder &e, core::SymbolRef litSymbol, bool definesBehavior) {
         auto definedHereLoc = litSymbol.loc(ctx);
         if (definedHereLoc.file().data(ctx).isRBI()) {
             e.addErrorSection(
                 core::ErrorSection(core::ErrorColors::format("Consider marking this RBI file `{}` if it is meant to "
                                                              "declare unpackaged constants",
                                                              "# packaged: false"),
                                    {core::ErrorLine(definedHereLoc, "")}));
-        } else {
+        } else if (definesBehavior) {
             e.addErrorLine(definedHereLoc, "Defined here");
+        } else {
+            e.addErrorSection(core::ErrorSection(
+                core::ErrorColors::format("`{}` does not define behavior and thus will not be automatically exported",
```

Comment:
Should we add color here to explain that they will need to add the export manually?